tests/conftest.py

```diff
@@ -335,13 +336,24 @@ def verify_cleanup(
 
     for handle in event_loop._scheduled:  # type: ignore[attr-defined]
         if not handle.cancelled():
-            if expected_lingering_timers:
-                _LOGGER.warning("Lingering timer after test %r", handle)
-            elif handle._args and isinstance(job := handle._args[0], HassJob):
-                pytest.fail(f"Lingering timer after job {repr(job)}")
-            else:
-                pytest.fail(f"Lingering timer after test {repr(handle)}")
-            handle.cancel()
+            arepr = reprlib.aRepr
```

Comment:
Optional: Given its 7 lines of setup, could be worth making a context manager out of this.

Reply:
f2d69f04c1a8ba20cd243eb6583051e2a33d46c0